compiler/src/toEC.ml

```diff
@@ -126,105 +126,86 @@ let leaks_lval pd = function
   | Lmem (_, x,e) -> leaks_e_rec pd [int_of_word pd (snd (add_ptr pd (gkvar x) e))] e
 
 (* FIXME: generate this list automatically *)
+(* Adapted from EasyCrypt source file src/ecLexer.mll *)
 let ec_keyword = 
- [ "exact"
- ; "assumption"
- ; "smt"
- ; "by"
- ; "reflexivity"
- ; "done"
- ; "admit"
- ; "axiom"
- ; "axiomatized"
- ; "lemma"
- ; "realize"
- ; "choice"
- ; "proof"
- ; "qed"
- ; "goal"
- ; "end"
- ; "import"
- ; "export"
- ; "include"
- ; "local"
- ; "declare"
- ; "hint"
- ; "nosmt"
- ; "module"
- ; "of"
- ; "const"
- ; "op"
- ; "pred"
- ; "require"
- ; "theory"
- ; "abstract"
- ; "section"
- ; "type"
- ; "class"
- ; "instance"
- ; "print"
- ; "search"
- ; "as"
- ; "Pr"
- ; "clone"
- ; "with"
- ; "rename"
- ; "prover"
- ; "timeout"
- ; "why3"
- ; "dump"
- ; "Top"
- ; "Self"
- ; "time"
- ; "undo"
- ; "debug"
- ; "pragma"
+ [ "admit"
```

Comment:
I think there is a way to ask easycrypt to provide the list of KW.
This does not need to be done here but we should remember it.